web/modules/custom/portland_migrations/migrations/ppb_directives_categories_redirects.yml

```diff
@@ -0,0 +1,32 @@
+id: ppb_directives_categories_redirects
+label: 'Migrate legacy paths into Redirects table'
+langcode: en
+migration_group: ppb_directives
+source:
+  plugin: csv
+  path: modules/custom/portland_migrations/sources/ppb_directives_categories.csv
+  header_row_count: 1
+  constants:
+    redirect_prefix: 'internal:/taxonomy/term/'
+    uid_admin: 1
+    status_code: 301
+  keys:
+    - CATEGORY
+destination:
+  plugin: 'entity:redirect'
+process:
+  uid: constants/uid_admin
+  status_code: constants/status_code
+  temp_nid:
+    plugin: migration_lookup
+    source: CATEGORY
+    migration: ppb_directives_categories
+  redirect_source:
+    plugin: substr
+    source: URL
+    start: 1
+  redirect_redirect:
+    plugin: concat
+    source:
+      - constants/redirect_prefix
+      - "@temp_nid"
```

Comment:
This should  have a newline. Please follow up with another PR to fix. I'm merging this as is for demonstration purposes.